src/features/breadcrumbs/store.ts

```diff
@@ -0,0 +1,50 @@
+import { BreadcrumbElement } from 'pages/api/breadcrumbs';
+import { Action, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { remoteItem, RemoteItem } from 'utils/storeUtils';
+
+type BreadcrumbItem = {
+  elements: BreadcrumbElement[];
+  id: string;
+};
+
+export interface BreadcrumbsStoreSlice {
+  crumbsByPath: Record<string, RemoteItem<BreadcrumbItem>>;
+}
+
+const initialState: BreadcrumbsStoreSlice = {
+  crumbsByPath: {},
+};
+
+function isUpdatedAction(action: Action<string>) {
+  return action.type.endsWith('Updated');
+}
+
+const breadcrumbsSlice = createSlice({
+  extraReducers: (builder) => {
+    builder.addMatcher(isUpdatedAction, (state) => {
+      // In lieu of a general-purpose way of identifying what changed, when
+      // anything is updated, we invalidate all breadcrumbs.
```

Comment:
Would this be something to mark as a "TODO" comment? Or is the need for this type of identification noted somewhere else?

Reply:
I don't consider this a TODO (because I don't think it needs to be done). It wouldn't be bad to have it, but I'm not sure how we would solve it so the cost/benefit for me means I don't think we need to do anything about it.